helm_deploy/court-case-service/templates/_envs.tpl

```diff
@@ -115,6 +115,42 @@ env:
         name: hmpps-domain-events-topic
         key: topic_arn
 
+  - name: HMPPS_SQS_QUEUES_PICPROBATIONOFFENDEREVENTSQUEUE_ACCESS_KEY_ID
+    valueFrom:
```

Comment:
This won't be tested by the build so we should check the deployment works manually before merge.